test/getGrist.ts

```diff
@@ -188,6 +197,12 @@ export class GristUtils extends GristWebDriverUtils {
     await this.waitForServer();
   }
 
+  public async closeDoc(): Promise<void> {
+    await driver.get(this.url);
+    await this.waitForServer();
```

Comment:
Does this line do anything? The line above it navigates to a URL in the browser, but doesn't wait for the page to load, so I'm not sure `waitForServer()` will actually be able to monitor network activity yet.